test/UnitTest/Microsoft.AspNet.OData.Test.Shared/OpenComplexTypeTest.cs

```diff
@@ -215,7 +317,9 @@ private static IList<OpenCustomer> CreateCustomers()
                     {
                         Street = "Street " + i,
                         City = "City " + i,
-                        DynamicProperties = new Dictionary<string, object> { { "IntProp", IntValues[i] } }
+                        DynamicProperties = new Dictionary<string, object> { { "IntProp", IntValues[i] } },
+                        // Leaving LineA as null
```

Comment:
>// Leaving LineA as null [](start = 24, length = 24)

It's better to assign "null" if you explicitly want and add the comments after the assignment.